packages/protocol/scripts/sourcify-publish.ts

```diff
@@ -33,42 +33,26 @@ import Web3 from 'web3'
  *
  */
 
-interface BuildOptions {
-  network: string
-  buildArtifactsPath: string
-  proposalPath: string
+const _buildTargets = {
```

Comment:
Why do we need to rely on `typeof` instead this remaining an `interface`?

Reply:
I saw that in the build.ts it was using typeof instead of the interface and I thought that you preferred that way